src/components/DappBrowser/Homepage.tsx

```diff
@@ -114,11 +128,11 @@ const Card = ({ site, showMenuButton }: { showMenuButton?: boolean; site: Trendi
                 />
               </Box>
               <Stack space="10px">
-                <Text size="17pt" weight="heavy" color="label">
+                <Text size="17pt" weight="heavy" color="label" numberOfLines={2}>
                   {site.name}
                 </Text>
-                <Text size="13pt" weight="bold" color="labelTertiary">
-                  {site.url}
+                <Text size="13pt" weight="bold" color="labelTertiary" numberOfLines={1}>
+                  {site.url.startsWith('http:') || site.url.startsWith('https') ? getDappHost(site.url) : site.url}
```

Comment:
could just do `site.url.startsWith('http')` right?

Reply:
`httpwebsite.com`  is a valid domain
```suggestion
                  {site.url.startsWith('http:') || site.url.startsWith('https:') ? getDappHost(site.url) : site.url}
```